enterprise/system-requirements.md

```diff
@@ -24,14 +24,19 @@ JDK 8
 
 ### Java (J2EE) Application Servers
 
-* Eclipse Jetty 9.4 or later
+* Eclipse Jetty:
+    - 10
+    - 9.4+
 * WebSphere Application Server (WAS) 8 or later
 * Apache Tomcat:
-    - 9 or later
+    - 10 (See note below)
+    - 9+
     - 8.5.12+
     - 8.0.42+
     - 7.0.76+
 
+> **Note:** Tomcat 10 will require WAR files to be places in *webapp-javaee* rather than *webapps* due to the change to Jakarta servlets. Jetty 11 is not currently supported due to this change.
```

Comment:
```suggestion
> **Important:** Tomcat 10 will require WAR files to be placed in the *webapp-javaee* directory, rather than the *webapps* directly, due to the change to Jakarta servlets. Jetty 11 is not currently supported due to this change.
```